iGCS/UserInterface/ViewControllers/DefaultSettingsTableViewController.m

```diff
@@ -0,0 +1,205 @@
+    
+    static NSString *cellIdentifier = @"Cell";
+    NSString *key = [self.sectionKeysArray objectAtIndex:indexPath.section];
+    NSArray *contents = [self.sectionContentsDict objectForKey:key];
+    NSString *cellContent = [contents objectAtIndex:indexPath.row];
+    
+    
+    UITableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:cellIdentifier];
+    if (cell == nil) {
+        cell = [[UITableViewCell alloc] initWithStyle:UITableViewCellStyleDefault reuseIdentifier:cellIdentifier];
+    }
+    
+    cell.textLabel.text = cellContent;
+    
+    if ([cellContent isEqual: @"Units"]) {
+        SettingsSegementedControlCell *switchView = [[SettingsSegementedControlCell alloc] initWithFrame:CGRectZero];
+        cell.accessoryView = switchView;
+    }
+    else {
+        cell.accessoryType = UITableViewCellAccessoryDisclosureIndicator;
+    }
+    
+    return cell;
+}
+
+//Not sure I like the header, commend out for now
+
+-(NSString *)tableView:(UITableView *)tableView titleForHeaderInSection:(NSInteger)section {
+    return [self.sectionKeysArray objectAtIndex:section];
+}
+
+
+
+#pragma mark - Table view delegate
+
+- (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath {
+
+    [tableView deselectRowAtIndexPath:indexPath animated:YES];
+    NSString *key = [self.sectionKeysArray objectAtIndex:indexPath.section];
+    NSArray *contents = [self.sectionContentsDict objectForKey:key];
+    NSString *cellContent = [contents objectAtIndex:indexPath.row];
+    
+    if ([cellContent isEqual: @"Waypoints"]) {
+        WaypointSettingsViewController *wayPointSettingsViewController = [[WaypointSettingsViewController alloc] initWithStyle:UITableViewStyleGrouped];
+        [self.navigationController pushViewController:wayPointSettingsViewController animated:YES];
+    }
+    
+    
+}
+
+
+/* Probably eventually need to implement this to customize our header
+- (UIView *)tableView:(UITableView *)tableView viewForHeaderInSection:(NSInteger)section {
+    
+    UILabel *myLabel = [[UILabel alloc] init];
+    myLabel.frame = CGRectMake(20, 8, 320, 20);
+    myLabel.font = [UIFont boldSystemFontOfSize:18];
+    myLabel.text = [self tableView:tableView titleForHeaderInSection:section];
+    
+    UIView *headerView = [[UIView alloc] init];
+    [headerView addSubview:myLabel];
+    
+    return headerView;
+}
+*/
+
+
+#pragma mark - UINavigationBar Button handlers
+
+- (void)cancelChanges:(id)sender {
+    
+    [self dismissViewControllerAnimated:YES completion:nil];
+}
+
+- (void)saveSettings:(id)sender {
+    DDLogDebug(@"Save Default Settings");
+    
+}
+
+- (void)didReceiveMemoryWarning {
+    [super didReceiveMemoryWarning];
+    // Dispose of any resources that can be recreated.
+}
+
+
+- (void) switchChanged:(id)sender {
+    UISwitch *switchControl = sender;
+    NSLog(@"The switch is %@", switchControl.on ? @"Standard" : @"Metric");
```

Comment:
Can you change this and the other log statements to `DDLogVerbose` where appropriate. DDLogInfo is also an option.